### PR TITLE
Week 11 day 01 mod 01

### PR DESCRIPTION
**Pull request by cameronjacoby**

### CODE REVIEW

**Comment by cameronjacoby** [comment]
@ben-manning @eerwitt - This is no longer WIP. Ready for review.

File: week-11-front-end-frameworks-and-mobile/day-01/module-01/readme.md [hunk 1]
```diff
@@ -1 +1,152 @@
 # <img src="https://cloud.githubusercontent.com/assets/7833470/10899314/63829980-8188-11e5-8cdd-4ded5bcb6e36.png" height="60"> Angular Auth with Satellizer
+
+| Objectives |
+| :--- |
+| Compare and contrast Cookie-Session and Web Token authentication strategies |
+| Use the Satellizer module to implement Web Token authentication in Angular |
+
+## Background
+
+With Express and Rails you learned the Cookie-Session authentication strategy. However, there's a better way to communicate authentication in **Single Page Applications**. You're going to use an encrypted chunk of JSON called a **JSON Web Token**, or JWT (pronounced *"jot"*), to communicate authentication between client and server.
```
Comment (eerwitt) [inline week-11-front-end-frameworks-and-mobile/day-01/module-01/readme.md]: It is minor but it'd be more accurate to say:

> ... use an encoded and signed chunk of ...

**Comment by eerwitt** [comment]
Looks great, solution is well put together also. :+1:

**Comment by cameronjacoby** [comment]
@eerwitt Thanks for the review! I fixed the wording on the JWT explanation.

**Pull request merged**